corehq/ex-submodules/casexml/apps/case/tests/test_indexes.py

```diff
@@ -61,17 +61,24 @@ class IndexTest(TestCase):
     MOTHER_CASE_ID = 'text-index-mother-case'
     FATHER_CASE_ID = 'text-index-father-case'
 
-    def setUp(self):
-        super(IndexTest, self).setUp()
+    @classmethod
+    def setUpClass(cls):
+        super(IndexTest, cls).setUpClass()
         delete_all_users()
-        self.project = Domain(name='index-test')
-        self.project.save()
-        self.user = create_restore_user(domain=self.project.name)
+        cls.project = Domain(name='index-test')
+        cls.project.save()
+        cls.user = create_restore_user(domain=cls.project.name)
 
     def tearDown(self):
         FormProcessorTestUtils.delete_all_cases()
         super(IndexTest, self).tearDown()
 
+    @classmethod
+    def tearDownClass(cls):
+        delete_all_users()
```

Comment:
this fails for me often on travis because there are other tests in the casexml suite that don't clean up their users properly unfortunately